app/services/analytics.rb

```diff
@@ -97,14 +97,17 @@ def browser_attributes
   IAL2_RECOVERY = 'IAL2 Recovery'.freeze # visited or submitted is appended
   IAL2_RECOVERY_REQUEST = 'IAL2 Recovery Request'.freeze
   IAL2_RECOVERY_REQUEST_VISITED = 'IAL2 Recovery Request Visited'.freeze
+  IAL2_RECOVERY_REQUEST_RATE_LIMIT_TRIGGERED = 'IAL2 Recovery Request Rate Limited'.freeze
```

Comment:
all the other events have lowercase `rate limited`, can we do that here too? Since cloudwatch regexes are case-sensitive by default it would make it easier for us to find all the rate limits with one query if we wanted to
```suggestion
  IAL2_RECOVERY_REQUEST_RATE_LIMIT_TRIGGERED = 'IAL2 Recovery Request rate limited'.freeze
```

Reply:
Yeah, this was an unfortunate choice between consistency amongst the peer of "IAL2 Recovery Request" events vs. "rate limited" events. You raise a good point about being able to query rate limited events.

I was actually beginning to explore an alternative approach where the logging happens as part of the throttling behaviors (e.g. in or as a result of `Throttler::IsThrottledElseIncrement` and `Throttler::Increment`). To your point, this would have the added benefit of there being only one event associated with rate limits being triggered.